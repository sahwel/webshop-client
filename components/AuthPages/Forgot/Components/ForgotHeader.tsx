import React from 'react';

const ForgotHeader = () => {
  return <h1 className="text-[300%]">Create your new password</h1>;
};

export default ForgotHeader;
