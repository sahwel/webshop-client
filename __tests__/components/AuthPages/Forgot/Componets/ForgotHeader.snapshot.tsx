import { render, renderHook, RenderHookResult } from '@testing-library/react';
import ForgotHeader from '@/components/AuthPages/Forgot/Components/ForgotHeader';

describe('ForgotHeader Snapshot', () => {
  it('renders ForgotHeader unchanged', () => {
    const { container } = render(<ForgotHeader />);
    expect(container).toMatchSnapshot();
  });
});
