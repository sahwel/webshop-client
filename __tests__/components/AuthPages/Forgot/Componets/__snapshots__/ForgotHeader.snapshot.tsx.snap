// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotHeader Snapshot renders ForgotHeader unchanged 1`] = `
<div>
  <h1
    class="text-[300%]"
  >
    Create your new password
  </h1>
</div>
`;
